contentcuration/contentcuration/frontend/channelList/views/Channel/StudioMyChannels.vue

```diff
@@ -0,0 +1,533 @@
+<template>
+
+  <div
+    class="my-channels"
+    :class="{
+      'studio-small': windowIsSmall,
+      'studio-medium': windowIsMedium,
+      'studio-large': windowIsLarge,
+      ['windowBreakpoint-' + windowBreakpoint]: windowBreakpoint,
+    }"
+  >
+    <div class="new-channel">
+      <KButton
+        v-if="!loading"
+        primary
+        data-test="add-channel"
+        :text="$tr('channel')"
+        @click="newChannel"
+      />
+    </div>
+    <div class="channels-body">
+      <p
+        v-if="!listChannels.length && !loading"
+        class="no-channels"
+      >
+        {{ $tr('noChannelsFound') }}
+      </p>
+      <KCardGrid
+        layout="1-1-1"
+        :loading="loading"
+        :skeletonsConfig="[
+          {
+            breakpoints: [0, 1, 2, 3, 4, 5, 6, 7],
+            orientation: 'vertical',
+            count: 3,
+          },
+        ]"
+        class="cards"
+      >
+        <KCard
+          v-for="(channel, index) in listChannels"
+          :key="channel.id"
+          :headingLevel="2"
+          thumbnailDisplay="small"
+          :thumbnailSrc="thumbnailSrc(channel)"
+          :thumbnailAlign="'left'"
+          :thumbnailScaleType="'fitXY'"
+          :orientation="windowIsSmall ? 'vertical' : 'horizontal'"
+          :title="channel.name"
+          :titleMaxLines="2"
+          :data-testid="`card-${index}`"
+          @click="goToChannelRoute(channel)"
+        >
+          <template #thumbnailPlaceholder>
+            <KIcon
+              :style="{ fontSize: '48px' }"
+              icon="image"
+            />
+          </template>
+          <template #belowTitle>
+            <div class="cards-title">
+              <div class="cards-resource">
+                <span> {{ $tr('resourceCount', { count: channel.count || 0 }) }} </span>
+                <span>
+                  {{ language(channel) }}
+                </span>
+              </div>
+              <div
+                class="cards-desc"
+                :style="{ color: $themePalette.black }"
+              >
+                {{ channel.description }}
+              </div>
+            </div>
+          </template>
+          <template #footer>
+            <div class="footer">
+              <div class="footer-left">
+                <span :style="{ color: $themePalette.grey.v_700 }">
```

Comment:
```suggestion
                <span :style="{ color: $themeTokens.annotation }">
```

Same reason as in the other similar comment.

https://design-system.learningequality.org/colors#tokens-annotation